docs/plugins/testing.md

```diff
@@ -276,9 +276,9 @@ export {
 }
 ```
 
-**./Api.test.js**
+**`./MyApi.test.js`**
 
-```
+```js
 /* eslint-disable import/first */
 
 jest.mock('./MyApi'); // Instruct Jest to swap all future imports of './MyApi.js' to './__mocks__/MyApi.js'
```

Comment:
based on this statement, assumed the example was intending to use these filenames, so updated above